Using base prefix '/usr'
New python executable in /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-data-model/venv/bin/python3
Not overwriting existing python script /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-data-model/venv/bin/python (you must use /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-data-model/venv/bin/python3)
Installing setuptools, pip, wheel...done.
Running virtualenv with interpreter /home/ptisnovs/bayesian/fabric8-analytics-common/dashboard/venv/bin/python3
Requirement already satisfied: pyflakes in ./venv/lib/python3.6/site-packages (2.0.0)
----------------------------------------------------
Checking source files for common errors in following
directories:
src test
----------------------------------------------------

src/graph_populator.py
    Pass
src/__init__.py
    Pass
src/utils.py
src/utils.py:9: 'os' imported but unused
src/utils.py:43: undefined name 'response'
    Fail
src/rest_api.py
src/rest_api.py:23: undefined name 'reload'
    Fail
src/data_source/__init__.py
    Pass
src/data_source/abstract_data_source.py
    Pass
src/data_source/s3_data_source.py
    Pass
src/graph_manager.py
    Pass
src/config.py
    Pass
src/data_importer.py
    Pass
test/test_rest_api.py
    Pass
test/test_graph_properties.py
    Pass
test/test_postgres_handler.py
    Pass
test/test_insertion_from_minio.py
    Pass
test/test_init.py
    Pass
test/test_data_importer.py
    Pass
test/test_s3_data_source.py
    Pass
test/test_graph_manager.py
    Pass
test/conftest.py
    Pass
test/test_graph_populator.py
    Pass
----------------------------------------------------
Checking following source files for common errors:
sanitycheck.py
----------------------------------------------------

sanitycheck.py
    Pass
2 source files out of 21 files needs to be checked and fixed
